tests/ops/functions/test_is_commuting.py

```diff
@@ -204,32 +187,11 @@ def z():
     def test_rx_mcz(self, wires, res):
         """Commutation between RX and MCZ"""
 
-        def z():
-            qml.PauliZ(wires=wires[1][2])
-
-        commutation = qml.is_commuting(
-            qml.RX(0.1, wires=wires[0][0]), qml.ctrl(z, control=wires[1][:-1])()
-        )
-        assert commutation == res
-
-    @pytest.mark.parametrize(
-        "wires,res",
-        [
-            ([[0], [0, 1, 2]], True),
-            ([[1], [0, 1, 2]], True),
-            ([[2], [0, 1, 2]], True),
-        ],
-    )
-    def test_mcz_rx(self, wires, res):
-        """Commutation between MCZ and RZ"""
```

Comment:
Why did you delete this test (`MCZ` and `RZ`)?

Reply:
I combined `test_rx_mcz` and `test_mcz_rx` together.  We now test both orders in the same test.